pkg/core/app/app.go

```diff
@@ -111,6 +111,14 @@ func (a *App) SetupDocker() error {
 		}
 	}
 
+	a.logger.Debug("inside setup docker", zap.String("cmd", a.cmd))
+
+	if RuntimeHooks != nil {
+		a.cmd, _ = RuntimeHooks.BeforeDockerSetup(a.logger, a.cmd)
```

Comment:
Don't ignore the error, in oss we won't get any error but still we should not ignore it.